discovery/mdns.md

```diff
@@ -85,6 +85,8 @@ The **additional records** of the response contain the peer's discovery details:
 
 The TXT record contains the multiaddresses that the peer is listening on. Each multiaddress is a TXT attribute with the form `dnsaddr=/.../p2p/QmId`. Multiple `dnsaddr` attributes and/or TXT records are allowed.
 
+Note that `mDNS` requires link-local addresses. Loopback and "NAT busting" addresses should not be sent and must be ignored on receipt.

```

Comment:
Does it really? Where does it say so?

As I've mentioned in my previous review, it is useful to include localhost addresses. Note that we can put into the TXT record whatever we deem useful, no matter what the mDNS RFC says.

Reply:
I take your point then.